tsl/src/remote/txn_resolve.c

```diff
@@ -22,10 +22,21 @@ remote_txn_resolution(Oid foreign_server, const RemoteTxnId *transaction_id)
 		/* transaction still ongoing; don't know it's state */
```

Comment:
GitHub doesn't let me comment on the above line...
`remote_txn_is_still_in_progress` looks weird -- it actually checks whether the access node transaction id (`RemoteTxnId.xid`) is in current MVCC snapshot. I'm not sure what this even means. I remember we discussed removing this snapshot check, is it still needed here?

Reply:
If it is in the current MVCC snapshot, it means the other transaction is still ongoing (not committed) so no decision on commit or abort has been made at this point. Thus, the healing function should ignore that GID.

Reply:
@akuzm agreed. I have renamed the function and added a comment on top as well.